### ISSUE TITLE
Error while toggling off Hide Overlays

### ISSUE THREAD

**Issue by jlampel** [CGCookie/retopoflow#1190]
I'm running into an error when turning off Hide Overlays. It doesn't appear to matter what other options are enabled at the time. 

Automatically Generated Info
-----------------------------------

```
Environment:

- RetopoFlow: 3.3.0α
- RF git: v3.3.0 df2e6dfa6f9222dcd3eb92e2844bcbf71cccafc4
- Blender: 3.04.1 blender-v3.4-release 2022-12-19
- Platform: Windows, 10, 10.0.18363, AMD64, AMD64 Family 23 Model 1 Stepping 1, AuthenticAMD
- GPU: NVIDIA Corporation, NVIDIA GeForce RTX 3060/PCIe/SSE2, 4.5.0 NVIDIA 528.24, (no shader version)
- Timestamp: 2023-02-14 14:15:36.068751
- Undo: tweak move, tweak move, tweak move, select all, select all, select all, select all, select all, select all, select all

Error Hash: e741f4effdc4e6e4eb82d7abe5a5904f

Trace:

- EXCEPTION (<class 'AttributeError'>): 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
-     C:\Program Files\Blender Foundation\Blender 3.4\3.4\scripts\modules\bpy_types.py
- 000 0816:__getattribute__() return super().__getattribute__(attr)
-     <string>
- 001 0055:update_hide_overlays() 
- 002 0001:<module>() 
-     .../common\utils.py
- 003 0197:run_it() return exec(action, f_globals, nf_locals)
- 004 0202:run_it() raise e
-     .../common\ui_core.py
- 005 2069:_fire_event() cb(details)
- 006 2075:_fire_event() raise e
- 007 2106:dispatch_event() self._fire_event(event, ui_event)
-     .../common\ui_properties.py
- 008 1049:_checked_change() self.dispatch_event('on_input')
-     .../common\boundvar.py
- 009 0074:_call_callbacks() for cb in self._callbacks: cb()
- 010 0101:set_value() self._call_callbacks()
- 011 0106:value() def value(self, value): self.set_value(value)
-     .../common\ui_properties.py
- 012 1039:checked() self._checked.value = v
-     <string>
- 013 0001:<module>() 
-     .../common\utils.py
- 014 0197:run_it() return exec(action, f_globals, nf_locals)
- 015 0202:run_it() raise e
-     .../common\ui_core.py
- 016 2069:_fire_event() cb(details)
- 017 2075:_fire_event() raise e
- 018 2106:dispatch_event() self._fire_event(event, ui_event)
-     .../common\ui_elements.py
- 019 0726:mouseclick() ui_for.dispatch_event('on_mouseclick')
-     .../common\ui_core.py
- 020 2065:_fire_event() cb(details)
- 021 2075:_fire_event() raise e
- 022 2106:dispatch_event() self._fire_event(event, ui_event)
-     .../common\ui_document.py
- 023 0555:mousedown_exit() under_mouseclick.dispatch_event('on_mouseclick')
-     .../common\fsm.py
- 024 0169:wrapped() return fn(*args, **kwargs)
```

Console Log:
------------------------------

```
RetopoFlow: Setting up symmetry data structures (0.0073697566986083984)
RetopoFlow: Setting up rotation target (0.0073699951171875)
RetopoFlow: Setting up RetopoFlow states (0.015793323516845703)
RetopoFlow: Setting up RetopoFlow tools (0.0094757080078125)
RetopoFlow: Setting up grease marks (0.010527849197387695)
RetopoFlow: Setting up visualizations (0.011580944061279297)
RetopoFlow: Setting up user interface (0.005264997482299805)
RetopoFlow: Setting up undo system (0.6833114624023438)
RetopoFlow: Checking auto save / save (0.06317353248596191)
RetopoFlow: Checking target symmetry (0.0221102237701416)
RetopoFlow: Loading welcome message (0.014741182327270508)
RetopoFlow: Resuming help image preloading (0.015793561935424805)
RetopoFlow: done with start
Caught exception while trying to run a delay_exec
  action: update_hide_overlays(self)
  except: 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
COOKIE CUTTER >> Exception Caught while trying to callback event handlers
UI_Element: <input type="checkbox" title="If enabled, overlays (source wireframes, grid, axes, etc.) are hidden.">
event: on_input
exception: 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
Caught exception while trying to run a delay_exec
  action: self.checked = not bool(self.checked)
  except: 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
COOKIE CUTTER >> Exception Caught while trying to callback event handlers
UI_Element: <input type="checkbox" title="If enabled, overlays (source wireframes, grid, axes, etc.) are hidden.">
event: on_mouseclick
exception: 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
COOKIE CUTTER >> Exception Caught while trying to callback event handlers
UI_Element: <label>
event: on_mouseclick
exception: 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
FSM: Caught exception in mousedown_exit (loc:C:\Users\jonat\AppData\Roaming\Blender Foundation\Blender\3.4\scripts\addons\retopoflow\addon_common\common\ui_document.py:528, state:"mousedown", substate:"exit")
     Exception: 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
____________________________________________________________________________________________________
Exception Info
- Time: 2023-02-14 14:16:58.064639
- Count: 5
- Hash: e741f4effdc4e6e4eb82d7abe5a5904f
- Info:
  - EXCEPTION (<class 'AttributeError'>): 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
  -     C:\Program Files\Blender Foundation\Blender 3.4\3.4\scripts\modules\bpy_types.py
  - 000 0816:__getattribute__() return super().__getattribute__(attr)
  -     <string>
  - 001 0055:update_hide_overlays()
  - 002 0001:<module>()
  -     .../common\utils.py
  - 003 0197:run_it() return exec(action, f_globals, nf_locals)
  - 004 0202:run_it() raise e
  -     .../common\ui_core.py
  - 005 2069:_fire_event() cb(details)
  - 006 2075:_fire_event() raise e
  - 007 2106:dispatch_event() self._fire_event(event, ui_event)
  -     .../common\ui_properties.py
  - 008 1049:_checked_change() self.dispatch_event('on_input')
  -     .../common\boundvar.py
  - 009 0074:_call_callbacks() for cb in self._callbacks: cb()
  - 010 0101:set_value() self._call_callbacks()
  - 011 0106:value() def value(self, value): self.set_value(value)
  -     .../common\ui_properties.py
  - 012 1039:checked() self._checked.value = v
  -     <string>
  - 013 0001:<module>()
  -     .../common\utils.py
  - 014 0197:run_it() return exec(action, f_globals, nf_locals)
  - 015 0202:run_it() raise e
  -     .../common\ui_core.py
  - 016 2069:_fire_event() cb(details)
  - 017 2075:_fire_event() raise e
  - 018 2106:dispatch_event() self._fire_event(event, ui_event)
  -     .../common\ui_elements.py
  - 019 0726:mouseclick() ui_for.dispatch_event('on_mouseclick')
  -     .../common\ui_core.py
  - 020 2065:_fire_event() cb(details)
  - 021 2075:_fire_event() raise e
  - 022 2106:dispatch_event() self._fire_event(event, ui_event)
  -     .../common\ui_document.py
  - 023 0555:mousedown_exit() under_mouseclick.dispatch_event('on_mouseclick')
  -     .../common\fsm.py
  - 024 0169:wrapped() return fn(*args, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
CookieCutter_Exceptions: handling caught exception
    action: handle exception caught by FSM
____________________________________________________________________________________________________
Exception Info
- Time: 2023-02-14 14:16:58.067797
- Count: 6
- Hash: e741f4effdc4e6e4eb82d7abe5a5904f
- Info:
  - EXCEPTION (<class 'AttributeError'>): 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
  -     C:\Program Files\Blender Foundation\Blender 3.4\3.4\scripts\modules\bpy_types.py
  - 000 0816:__getattribute__() return super().__getattribute__(attr)
  -     <string>
  - 001 0055:update_hide_overlays()
  - 002 0001:<module>()
  -     .../common\utils.py
  - 003 0197:run_it() return exec(action, f_globals, nf_locals)
  - 004 0202:run_it() raise e
  -     .../common\ui_core.py
  - 005 2069:_fire_event() cb(details)
  - 006 2075:_fire_event() raise e
  - 007 2106:dispatch_event() self._fire_event(event, ui_event)
  -     .../common\ui_properties.py
  - 008 1049:_checked_change() self.dispatch_event('on_input')
  -     .../common\boundvar.py
  - 009 0074:_call_callbacks() for cb in self._callbacks: cb()
  - 010 0101:set_value() self._call_callbacks()
  - 011 0106:value() def value(self, value): self.set_value(value)
  -     .../common\ui_properties.py
  - 012 1039:checked() self._checked.value = v
  -     <string>
  - 013 0001:<module>()
  -     .../common\utils.py
  - 014 0197:run_it() return exec(action, f_globals, nf_locals)
  - 015 0202:run_it() raise e
  -     .../common\ui_core.py
  - 016 2069:_fire_event() cb(details)
  - 017 2075:_fire_event() raise e
  - 018 2106:dispatch_event() self._fire_event(event, ui_event)
  -     .../common\ui_elements.py
  - 019 0726:mouseclick() ui_for.dispatch_event('on_mouseclick')
  -     .../common\ui_core.py
  - 020 2065:_fire_event() cb(details)
  - 021 2075:_fire_event() raise e
  - 022 2106:dispatch_event() self._fire_event(event, ui_event)
  -     .../common\ui_document.py
  - 023 0555:mousedown_exit() under_mouseclick.dispatch_event('on_mouseclick')
  -     .../common\fsm.py
  - 024 0169:wrapped() return fn(*args, **kwargs)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
CookieCutter_Exceptions._callback_exception_callbacks: 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
RF_UI.handle_exception: 'VIEW3D_OT_RetopoFlow_LastTool' object has no attribute 'overlays_restore'
Info: Saved "cobra_RetopoFlow_AutoSave_EMERGENCY.blend"
GitHub: Not reported, yet
RetopoFlow: unscaling from unit box
RetopoFlow: scaling to factor=1.0, view='RESTORE', clip='RESTORE', mesh='RESTORE'
RetopoFlow: unscaling from unit box
RetopoFlow: scaling to factor=1.0, view='RESTORE', clip='RESTORE', mesh='RESTORE'
```

**Comment by vxlcoder** [comment]
this issue should now be fixed in commit e486f54

**Comment by jlampel** [comment]
Works great now 👍
